ci/ci/github.py

```diff
@@ -414,12 +414,22 @@ def _hail_github_status_from_statuses(statuses_json) -> Dict[str, GithubStatus]:
         hail_statuses = {}
         for s in statuses:
             context = s['context']
-            if context == GITHUB_STATUS_CONTEXT or context.startswith('hail-ci'):
+            if context == 'ci-test' or context.startswith('hail-ci'):
                 if context in hail_statuses:
                     raise ValueError(
                         f'github sent multiple status summaries for context {context}: {s}\n\n{statuses_json}'
                     )
                 hail_statuses[context] = GithubStatus(s['state'])
+
+        # FIXME: This is a temporary workaround to change the name of
+        # the `ci-test` context to `hail-ci-gcp` without recomputing statuses
+        # for all PRs. Existing `ci-test` statuses should be treated as the new status
+        # from GCP, and new GCP statuses should take precedence over old `ci-test`
+        # statuses.
+        # Once all PRs have a `hail-ci-gcp` status on GitHub, this and the above
+        # special-casing of `ci-test` should be removed.
+        if 'ci-test' in hail_statuses and 'hail-ci-gcp' not in hail_statuses:
```

Comment:
Ya I think I don't want to deal with any CI hiccups at the moment, so I've dramatically simplified the situation. Now what we're saying is pay attention to any statuses that are either `ci-test` or starts with `hail-ci` (as in `hail-ci-azure`). GCP should behave exactly the same, since its `GITHUB_STATUS_CONTEXT` is `ci-test`, and now azure should see the status from GCP